src/main/java/im/juniq/racingcar/view/ResultView.java

```diff
@@ -0,0 +1,35 @@
+package im.juniq.racingcar.view;
+
+import java.util.List;
+
+import im.juniq.racingcar.domain.CarState;
+
+public class ResultView {
+	public void printScore(List<CarState> carStates) {
+		carStates.forEach(carState ->
+			System.out.println(carState.name() + " : " + positionGraph(carState.position())));
+		System.out.println();
+	}
+
+	private String positionGraph(int position) {
+		String positionGraph = "";
+		for (int i = 0; i < position; i++) {
+			positionGraph += "-";
+		}
+		return positionGraph;
+	}
```

Comment:
```suggestion
      private String positionGraph(int position) {
		StringBuilder positionGraph = new StringBuilder();
		for (int i = 0; i < position; i++) {
			positionGraph.append("-");
		}
		return positionGraph.toString();
	}
```

문자열을 붙이는 연산에서는 StringBuilder를 사용하길 권해드립니다.
String은 + 연산 이전에 문자열과 연산 이후의 연산 문자열이 각각 별개의 인스턴스로 존재합니다.
따라서 "----" 와 같은 문자열을 만드는 과정에서 ("", "-", "--", "---", "----") 5개의 인스턴스가 생기게 되어서 비효율적인 메모리 사용을 발생시킵니다.

--- 

혹시 아래 이슈들을 모르셨다면 한번 찾아보시길 권해드립니다. 😁 
Q) StringBuilder, StringBuffer의 차이 (면접 단골질문이더라구요)
Q) 자바 1.5버전 이상부터는 String 연산을 컴파일러가 알아서 최적화하여 굳이 StringBuilder를 안써도 된다고 하던데...? 진짜 그럴까요? 그럼 언제 StringBuilder를 사용해야할까요?